rpc/core/status.go

```diff
@@ -16,6 +16,20 @@ import (
 // hash, app hash, block height and time.
 // More: https://docs.tendermint.com/master/rpc/#/Info/status
 func Status(ctx *rpctypes.Context) (*ctypes.ResultStatus, error) {
+	var (
+		earliestBlockMeta     *types.BlockMeta
+		earliestBlockHash     tmbytes.HexBytes
+		earliestAppHash       tmbytes.HexBytes
+		earliestBlockTimeNano int64
+	)
+	earliestBlockHeight := blockStore.Base()
+	earliestBlockMeta = blockStore.LoadBlockMeta(earliestBlockHeight)
+	if earliestBlockMeta != nil {
```

Comment:
How could this be nil? Is it only if pruning races here? If so is it fine to just not print everything? Maybe we should consider having an atomic Base+Load command so we don't have to worry about pruning races between when we call .Base() and .LoadXXX() ?

Reply:
Good point, will add an atomic `LoadBase()`.